src/vopr_hub/SETUP.md

```diff
@@ -0,0 +1,180 @@
+# VOPR and VOPR Hub Setup
```

Comment:
```suggestion
# VOPR Hub Setup
* Follow these instructions to setup the VOPR Hub, along with several full-time VOPR simulators, on a dedicated testing machine.*
```

I updated the title to put the emphasis on `VOPR Hub`, since users don't have to follow these setup steps if they just want to run the VOPR themselves locally on their own machine.